### DISCUSSION TITLE
Welcome to docker-jdownloader-2 Discussions!

### DISCUSSION THREAD

**Discussion by jlesage** [jlesage/docker-jdownloader-2#154] [Announcements]
## 👋 Welcome!
Discussions has been enabled for this project.  This is a place to connect with other members of the community.

Here, you can:
  * Ask questions about:
    * How to setup the container.
    * How to use the application.
    * Problems you might with your specific setup.
    * etc.
  * Share ideas.
  * Engage with other community members.

Welcome others and be open-minded. Remember, this is a community we build together 💪.

**Comment by Astroviridae** [comment]
Hi, i have installed the docker some hours ago. I have the problem that the download speed is very low. Only 1Mb/s per connection. With the same hoster and the same download files on the desktop app i have full speed with 500Mbit/s.
The interesting thing is, that at the first 4 seconds i had full speed but now its very slow. Even with more connections, chunks and severall restarts of the program/docker.
Anyone know why this is?

**Reply by jlesage** [reply]
Is the desktop app running on the same host that runs the container ?  More details about your setup would be useful.  In general, a Docker container should not affect network performance.
